core/src/main/resources/defaults/blueprints/7.2.17/cdp-sdx-scalable.bp

```diff
@@ -0,0 +1,602 @@
+{
+  "tags":{
+    "shared_services_ready":true
+  },
+  "description":"7.2.17 - Scalable SDX template with Atlas, HMS, Ranger and other services they are dependent on.  Services like HDFS, HBASE, RANGER, HMS have HA",
+  "blueprint":{
+    "cdhVersion":"7.2.17",
+    "displayName":"scalable-datalake",
+    "hostTemplates":[
+      {
+        "cardinality":2,
+        "refName":"master",
+        "roleConfigGroupsRefNames":[
+          "zookeeper-SERVER-BASE",
+          "hbase-MASTER-BASE",
+          "ranger-RANGER_ADMIN-BASE",
+          "hdfs-GATEWAY-BASE",
+          "hive-GATEWAY-BASE",
+          "hbase-GATEWAY-BASE",
+          "solr-GATEWAY-BASE",
+          "kafka-GATEWAY-BASE",
+          "hdfs-NAMENODE-BASE",
+          "hdfs-FAILOVERCONTROLLER-BASE",
+          "hdfs-JOURNALNODE-BASE"
+        ]
+      },
+      {
+        "cardinality":2,
+        "refName":"gateway",
+        "roleConfigGroupsRefNames":[
+          "atlas-ATLAS_SERVER-BASE",
+          "ranger-RANGER_USERSYNC-BASE",
+          "ranger-RANGER_TAGSYNC-BASE",
+          "hive-HIVEMETASTORE-BASE",
+          "knox-KNOX_GATEWAY-BASE",
+          "solr-GATEWAY-BASE",
+          "hdfs-GATEWAY-BASE",
+          "kafka-GATEWAY-BASE",
+          "hive-GATEWAY-BASE",
+          "hbase-GATEWAY-BASE"
+        ]
+      },
+      {
+        "cardinality":3,
+        "refName":"core",
+        "roleConfigGroupsRefNames":[
+          "hdfs-DATANODE-BASE",
+          "kafka-KAFKA_BROKER-BASE",
+          "solr-SOLR_SERVER-BASE",
+          "hbase-REGIONSERVER-BASE",
+          "solr-GATEWAY-BASE",
+          "hdfs-GATEWAY-BASE",
+          "kafka-GATEWAY-BASE",
+          "hive-GATEWAY-BASE",
+          "hbase-GATEWAY-BASE"
+        ]
+      },
+      {
+        "cardinality":1,
+        "refName":"auxiliary",
+        "roleConfigGroupsRefNames":[
+          "zookeeper-SERVER-BASE",
+          "hdfs-JOURNALNODE-BASE"
+        ]
+      },
+      {
+        "cardinality":2,
+        "refName":"idbroker",
+        "roleConfigGroupsRefNames":[
+          "knox-IDBROKER-BASE"
+        ]
+      },
+      {
+        "cardinality":0,
+        "refName":"solrHG",
+        "roleConfigGroupsRefNames":[
+          "solr-SOLR_SERVER-EXTERNAL"
+        ]
+      },
+      {
+        "cardinality":0,
+        "refName":"storageHG",
+        "roleConfigGroupsRefNames":[
+          "hbase-REGIONSERVER-EXTERNAL",
+          "hdfs-DATANODE-EXTERNAL"
+        ]
+      },
+      {
+        "cardinality":0,
+        "refName":"kafkaHG",
+        "roleConfigGroupsRefNames":[
+          "kafka-KAFKA_BROKER-EXTERNAL"
+        ]
+      },
+      {
+        "cardinality":0,
+        "refName":"razHG",
+        "roleConfigGroupsRefNames":[
+        ]
```

Comment:
Good to know, that this works without any servicces being configured here (maybe because the service gets added via the RazProvider in code).

Will need to test: What happens to this HostGroup is RAZ is not enabled.

Reply:
This was tested.
When RAZ is disabled, provisioning was successful. There will be a failure if we try to scale it.